brownie/network/rpc.py

```diff
@@ -98,6 +111,7 @@ def launch(self, cmd: str, **kwargs: Dict) -> None:
             if web3.isConnected():
                 self._reset_id = self._current_id = self._snap()
                 _notify_registry(0)
+                self._time_offset = self._request("evm_increaseTime", [0])
```

Comment:
What is the purpose of this?

Reply:
As far as I could see, "evm_increaseTime" is the only way to get the time offset out of ganache. This is essentially `sleep 0 and return the offset` to make sure the `Rpc.time()` is in sync with ganache from the start. If we omit it, then `Rpc.time()` is equal to the current time until we use `Rpc.sleep()` the first time.